.github/workflows/test-unit.yml

```diff
@@ -21,7 +21,10 @@ jobs:
         with: { go-version: "${{ matrix.go }}" }
       - run: go version
       - run: make test-unit race=true
-      - uses: codecov/codecov-action@v1
+        if: matrix.os != 'ubuntu-latest'
+      - run: make test-unit race=true coverage=true
+        if: matrix.os == 'ubuntu-latest'
+      - uses: codecov/codecov-action@v3
```

Comment:
What's the reason not to run code coverage on other platforms? Codecov supports merging multiple uploads, in fact it's better because some tests may not run on all platforms.

Reply:
It's good practice to run tests cross platform making sure we don't break anything since we claim support for those platforms. No platform specific stuff today doesn't mean it can't be accidentally introduced.